nss-day-cohort-48/rare-api-boastful-bulldogs
# Comment view & url - cleaned up squiggles in other files

User can now see all comments per post. They can add a comment and also delete comments. 

Still need to work on only the logged-in user who can delete their own comments.

## Steps to Test
(need to pull Client as well)
1. git fetch --all
2. checkout to anm-api-comment
3. run the server
4. click on view all comments
5. add a comment and the page refreshes to show new comment at top
6. delete a comment and page refreshes to show it gone
7. go back button should take you back to the post you were viewing
